core/src/main/kotlin/io/fritz2/binding/validation.kt

```diff
@@ -26,13 +29,16 @@ abstract class Validator<D, M: WithSeverity, T> {
 
     abstract fun validate(data: D, metadata: T): List<M>
 
-    fun isValid(msg: M): Boolean {
+    open fun validPredicate(msg: M): Boolean {
```

Comment:
since this is the only information we need about a message, wouldn't it be a good idea to put just a function `isInvalid` in the interface and no property at all?